src/google/protobuf/compiler/cpp/cpp_helpers.cc

```diff
@@ -664,9 +664,9 @@ std::string UInt64ToString(const std::string& macro_prefix, uint64_t number) {
 std::string DefaultValue(const FieldDescriptor* field) {
   switch (field->cpp_type()) {
```

Comment:
Actually I think this helper function could be simplified even further and just be a one-liner that returns `DefaultValue(Options(), field)`.

Reply:
Agreed.

It looks like this method is not used by the protoc itself, hence the issue was not discovered earlues. I believe that some of our plugins triggered the issue.

I suggest merge this PR to fix the typo while I am looking for the proper fix in the plugin.